manifests/params.pp

```diff
@@ -169,13 +162,8 @@
       $bindir                 = pick($bindir, "/usr/lib/postgresql/${version}/bin")
       $datadir                = pick($datadir, "/var/lib/postgresql/${version}/main")
       $confdir                = pick($confdir, "/etc/postgresql/${version}/main")
-      if pick($service_provider, $facts['service_provider']) == 'systemd' {
```

Comment:
> Also no linux operating system without Systemd is supported, so we can drop the old code.

Technically Debian can still run without systemd and the previous code did handle that at essentially no cost.

Reply:
@Ramesh7 can you tell me if we can drop support for Debian without systemd as init system?

Reply:
I've at least changed the PR title. I'd like the final commit message (if it is merged) to also reflect it.